src/style.css

```diff
@@ -1,3 +1,80 @@
+:root {
+  --link-color: #000;
+  --link-active-color: #00acdc;
+}
+
+html {
+  font-size: 10px;
+  font-family: Roboto, Arial, sans-serif;
```

Comment:
Consider adding a fallback font for Roboto. This is to ensure text is displayed with a default font if the primary font fails to load. For example: `font-family: Roboto, 'Arial', sans-serif;`.